### DISCUSSION TITLE
Install Odyssey

### DISCUSSION THREAD

**Discussion by rdcron** [rfvgyhn/min-ed-launcher#145] [Q&A] [answered]
Hi, thanks for this project. I'm playing Elite on an apple silicon Mac using crossover/Steam and min-ed-launcher is working great. I'm returning to Elite after a long break and decided to buy Odyssey(through Steam), but it seems like you have to use the default launcher to get Odyssey to install. Is there a way to get min-ed-launcher to do that installation?  Included is a screenshot when I have /edo set in properties on Steam. /edh4 works fine but only opens Horizons. Thanks!

![Screenshot 2024-12-03 at 3 48 51 PM](https://github.com/user-attachments/assets/f69f183f-2e55-4da1-be57-0928a3cb4a88)

**Comment by rfvgyhn** [comment, marked as answer]
It's not recognizing that you own Odyssey. It looks like you logged in via Frontier auth (`/frontier name`). Have you linked your Steam account to your Frontier account? Was your original purchase through Steam or Frontier? I know that it used to be the case that you couldn't login with your Frontier credentials if you wanted to use your Steam license but I've read folks say that restriction has been lifted (haven't yet tried myself). I'm not sure how Frontier handles mix-and-match licenses.

I'm also not familiar with crossover so I don't know how it launches the game. Are you able to launch it through Steam or manually specify the `/steam` flag instead of `/frontier name`?

Posting the log file might provide some more insight. `users/crossover/AppData/Local/min-ed-launcher/min-ed-launcher.log`.

**Reply by rdcron** [reply]
Thanks, linking my Steam account to Frontier did the trick! Thanks so much!
